tests/full/test.sh

```diff
@@ -86,7 +86,8 @@ rlJournalStart
         [[ $PRE_RELEASE -ne 1 ]] && rlRun "sed 's/^Version:.*/Version: 9.9.9/' -i tmt.spec"
 
         # Build tmt packages
-        rlRun "dnf builddep -y tmt.spec" 0 "Install build dependencies"
+        rlRun "make develop" 0 "Install development packages"
```

Comment:
I believe it only needs `hatch` and `python3-devel` before running `make deps`.
What do you think about adding these two to `make deps` instead?

I'm also thinking whether it wouldn't be better to move the man page creation to spec file and put `python3-docutils` to BuildRequires. 
There are pros and cons, but not having to install hatch and let it create a single-use virtualenv for rst2man command might be more efficient.

Reply:
> What do you think about adding these two to `make deps` instead?

Hm, makes sense. So if you would like to build rpms you would just need `make deps && make rpms`, right?

Reply:
So, the answer seems to be `no` as `make deps` depends on `tarbal` which depends on `build` which needs `hatch` already installed.

Reply:
> Hm, makes sense. So if you would like to build rpms you would just need make deps && make rpms, right?

Yes

> So, the answer seems to be no as make deps depends on tarbal which depends on build which needs hatch already installed.

This is a response to the man page creation comment? 
I initially though `build` module is being installed alongside python3-devel, but it's not, so there has to be some build system installed anyway (using pip to create sdist is not recommended afaik).
It only make sense to have `hatch` and `hatch-vcs` required for rpm building (alongside `python3-devel`), as hatch rpm will also pull hatchling rpm, so backend, frontend and version control tools are taken from OS repos, rather than using PyPI versions, which could potentially be incompatible.

Reply:
> This is a response to the man page creation comment?

Ah, sorry for the confusion, that was back to the `make deps` target.

Reply:
Adjusted based on our discussion.